### PR TITLE
Cleanup Brocade SKU packages

### PR DESCRIPTION
**Pull request by dpasupathi**
Modified the conflicting skupack for brocade-vdx-6740 and added another skupack for brocade-vdx-6740T

### CODE REVIEW

**Review by anhou — APPROVED** [review]

File: brocade-VDX-6740/config.json [hunk 1]
```diff
@@ -1,9 +1,15 @@
...
-    "rules": [
-        {
-            "path": "snmp-1.ENTITY-MIB::entPhysicalDescr_1",
-            "equals": "VDX6740"
-        }
-    ]
+     "name": "Brocade VDX SKU",
+     "rules": [
+         {
+             "regex": "Hardware Rev                  : 131.7",
```
Comment (yyscamper) [inline brocade-VDX-6740/config.json]: So many spaces, it will be error-prone, please change to regex expression.

**Comment by yyscamper** [comment]
:+1: after fix comment.

**Comment by dpasupathi** [comment]
@yyscamper : Fixed the regex portion

File: brocade-VDX-6740/config.json [hunk 2]
```diff
@@ -2,7 +2,7 @@
      "name": "Brocade VDX SKU",
```
Comment (pscharla) [inline brocade-VDX-6740/config.json:2]: @dpasupathi : Just to make sure I've got it right, you are changing this file to be for generic "brocade-VDX"? Does this belong in the brocade-VDX skupack?

**Comment by brianparry** [comment]
ok to test

**Pull request merged**